src/ProvidedTypes.fs

```diff
@@ -4,1902 +4,1908 @@
 
 namespace ProviderImplementation.ProvidedTypes
 
-    #nowarn "1182"

```

Comment:
I think just undo all this change of indentation.  The indentation is there for a reason in any case